openlibrary/plugins/openlibrary/js/carousels/carousels.js

```diff
@@ -1,5 +1,22 @@
+// Slick#1.6.0 is not on npm
```

Comment:
For consistency with `editions-table` and the CSS, let's keep the name singular (so `carousel/carousel.js`)

Reply:
Done